integration-tests/plugin-helpers.mjs

```diff
@@ -0,0 +1,16 @@
+import http from 'http'
+
+export function onMessage (handler) {
+  const server = http.createServer(async (req, res) => {
+    try {
+      await handler()
```

Comment:
What happens first? The calling of `handler()` or the later `process.send()`? And is the order guaranteed?